spec/models/api_wrappers/kraken_api_wrapper_spec.rb

```diff
@@ -9,122 +9,104 @@
     BitexBot::Robot.setup
   end
 
-  def stub_public_client
+  def public_client_stub
```

Comment:
Why are we changing these method names? They are actions, not getters.
Here's an interpretation of both names:
 - stub_public_client: Stub the public client with a fake response
 - public_client_stub: Retrieve the stubbed response for the public client

Reply:
cleared, I had a bad interpretation.